wp-plugin-web.php

```diff
@@ -10,11 +10,12 @@
  * @wordpress-plugin
  * Plugin Name:       The Web.com Plugin
  * Plugin URI:        https://web.com
+ * Update URI:        https://github.com/newfold-labs/wp-plugin-web/
```

Comment:
@circlecube Should we update this to be https://github.com/newfold-labs/wp-plugin-web/blob/main/bootstrap.php#L89C19-L89C93?

Reply:
I can change these to point to the update uri, but from what reading the doc the url isn't actually used in any way. I figured if a user were to follow the URL having them hit the repo page might be better than the API endpoint which downloads the zip. See: https://make.wordpress.org/core/2021/06/29/introducing-update-uri-plugin-header-in-wordpress-5-8/